XML/TestCases/FunctionalTests.xml

```diff
@@ -801,4 +801,15 @@
         <Tags>cpu</Tags>
         <Priority>3</Priority>
     </test>
+    <test>
+        <testName>VALIDATE-DEPENDENCY-AGENT</testName>
+        <testScript>VALIDATE-DA.ps1</testScript>
```

Comment:
if no special thing to deal with in the ps1 file, then no need use ps1 file, put validate-da.sh here is enough, test framework will handle for you.

Reply:
That is correct. I'm not using ps1 file for anything special except for running my script as `runAsSudo`. Was wondering if I can achieve even that without the ps1 file ?

Reply:
close this comment because of no reply in 19 days.